src/layout/forceDirected.ts

```diff
@@ -106,12 +113,20 @@ export function forceDirected({
   const nodeStrengthAdjustment =
     is2d && edges.length > 25 ? nodeStrength * 2 : nodeStrength;
 
+  let forceX = d3ForceX(1200 / 2).strength(0.05);
```

Comment:
Since you are re-assigning - why not just make if/else to save on init of the object you throw away.